source/docs/03-metadata.xml

```diff
@@ -525,12 +543,7 @@
                            <specDesc key="biblStruct"/>
                         </specList>
                      </p>
-                     <div xml:id="headerAssociatingMetadataAndData" type="div5">
-                        <head>Associating Metadata and Data</head>
-                        <p>In the MEI header, the <att>data</att> attribute may be used to associate metadata with related notational elements.</p>
-                        <p>Similarly, in the body of the MEI document, the <att>decls</att> attribute may be used to associate parts of the encoded text with related metadata.</p>
-                        <p>The most useful associations of this type are between the bibliographic description of a source and the material taken from it.</p>
-                     </div>
+                     <p/>
```

Comment:
@pe-ro Was the deletion of this div about associating metadata and data intended?

Reply:
Sorry, just seeing that the change was commited by @riedde . @riedde , did you delete this section by intention?

Reply:
I would suggest to keep the div but to delete the last paragraph. Because I'm not sure if this is still the most useful association.

Reply:
ODD meeting: Proposal to move the section to 3.3 (Metadata concepts).

Reply:
Done in #1567